css/style.css

```diff
@@ -17,3 +17,119 @@ body {
  */
 
 
+/*Top Navigation*/
+html {
+    box-sizing: border-box;
+    font-size: 16px;
+    font-family: 'Roboto', sans-serif;
+  }
+  
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+  
+  
+  /*navigation*/
+  
+  .main-navigation {
+    background-color: #fff;
+    position: fixed;
+    width: 100%
+  }
+  /*styling the content*/
+  .main-navigation  .content {
+    width: 970px;
+    height: 4.375rem;
+    margin: 0 auto;
+    color: #838994;
+   
+  }
+  
+  .main-navigation .navigation {
+    float: right;
+    font-size: 1rem;
+    margin-top: 25px; 
+    -webkit-font-smoothing: antialiased;
+  }
+  
+  .main-navigation li {
+    display: inline-block;
+    margin-left: 35px;
+  }
+  
```

Comment:
Try re-sizing your page to a smaller size, does the navigation menu items move so that they are still visible? Have a think about how you could use a flexbox container to help the menu options flow with your page in your navigation class. 

Have look at adding some side padding as well so that there space around the elements when looking at smaller screens.